src/axes.jl

```diff
@@ -560,21 +589,26 @@ const _widen_seriestypes = (
     :scatter3d,
 )
 
-function default_should_widen(axis::Axis)
-    if axis[:widen] isa Bool
-        return axis[:widen]
-    end
+const default_widen_factor = 1.06
```

Comment:
Can we make this a default keyword argument to `widen_factor` ?

Reply:
I'm not sure how that could work, since `widen_factor` takes the axis as argument and chooses whether to use the default based on its `:widen` and `:lims` attributes.

Reply:
Never mind, I think I understand now what you mean - `widen_factor(axis::Axis; default_widen_factor = 1.06)`?
The problem is you might want to do `scale_lims(..., default_widen_factor)` as is done now in the `bar` recipe. So the default factor would need be specified again there or in the methods of `scale_lims`.

Reply:
I see. I missed that. Still it would be good to have
```julia
widen_factor(axis::Axis; factor = default_widen_factor)
```
then, IMO.

Reply:
done